app/src/main/java/com/github/polypoly/app/ui/game/HeadsUpDisplay.kt

```diff
@@ -335,18 +363,16 @@ fun ToggleIconButton(
         shape = CircleShape,
     )
     {
-        if (toggle) {
-            Image(
-                painter = painterResource(onIcon),
-                contentDescription = "Expand",
-                modifier = Modifier.size(50.dp)
-            )
-        } else {
-            Image(
-                painter = painterResource(offIcon),
-                contentDescription = "Collapse",
-                modifier = Modifier.size(50.dp)
-            )
-        }
+        if (toggle)
+            Icon(onIcon, contentDescription = null, modifier = Modifier.size(20.dp))
+        else
+            Icon(offIcon, contentDescription = null, modifier = Modifier.size(50.dp))
     }
 }
+
+/**
+ * Changes the opacity of a color
+ */
+fun changeOpacity(color: Color, opacity: Float): Color {
+    return Color(color.red, color.green, color.blue, opacity)
```

Comment:
you can use `color.copy(alpha = opacity)`

Reply:
fixed